### PR TITLE
Keras Import Fix - Embedding + Convolution

### LINKED ISSUE #7013: Error importing keras model
When trying to import the following keras model: (note conv2d_1 input shape (20,300,1)
image

The Kerasimporter in dl4j reads the input shuffled; (h=300,w=1,c=20)
I get the following error:

> Exception in thread "main" org.deeplearning4j.exception.DL4JInvalidConfigException: Invalid configuration for layer (idx=-1, name=conv2d_1, type=ConvolutionLayer) for width dimension:  Invalid input configuration for kernel width. Require 0 < kW <= inWidth + 2*padW; got (kW=300, inWidth=1, padW=0)
> Input type = InputTypeConvolutional(h=300,w=1,c=20), kernel = [6, 300], strides = [1, 1], padding = [0, 0], layer size (output channels) = 100, convolution mode = Truncate
> 	at org.deeplearning4j.nn.conf.layers.InputTypeUtil.getOutputTypeCnnLayers(InputTypeUtil.java:335)
> 	at org.deeplearning4j.nn.conf.layers.ConvolutionLayer.getOutputType(ConvolutionLayer.java:184)
> 	at org.deeplearning4j.nn.modelimport.keras.layers.convolutional.KerasConvolution2D.getOutputType(KerasConvolution2D.java:150)
> 	at org.deeplearning4j.nn.modelimport.keras.KerasModel.inferOutputTypes(KerasModel.java:304)
> 	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:179)
> 	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:96)
> 	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:307)
> 	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:144)
> 	at dl4jTrial.main(dl4jTrial.java:46)

Version Information

    Deeplearning4j version: 1.0.0-beta3
    platform information: Linux CentOS

Attached is the model
[model.h5.tar.gz](https://github.com/deeplearning4j/deeplearning4j/files/2760582/model.h5.tar.gz)

### PR DESCRIPTION
**Pull request by farizrahman4u**
#7013

### CODE REVIEW

**Comment by farizrahman4u** [comment]
Test file:


[model.zip](https://github.com/deeplearning4j/deeplearning4j/files/3060988/model.zip)

**Comment by maxpumperla** [comment]
@farizrahman4u next time, can you please add your model to test resources (if possible, a small version of it) and add a test? thanks
